pkg/apis/sources/v1alpha1/register.go

```diff
@@ -35,4 +40,21 @@ var (
 
 	// SchemeBuilder is used to add go types to the GroupVersionKind scheme
 	SchemeBuilder = &scheme.Builder{GroupVersion: SchemeGroupVersion}
+
+	AddToScheme = SchemeBuilder.AddToScheme
 )
+
+// Resource takes an unqualified resource and returns a Group qualified GroupResource
+func Resource(resource string) schema.GroupResource {
+	return SchemeGroupVersion.WithResource(resource).GroupResource()
+}
+
+// Adds the list of known types to Scheme.
+func addKnownTypes(scheme *runtime.Scheme) error {
```

Comment:
Why is this function necessary? It's not present in https://github.com/kubernetes-sigs/kubebuilder/blob/804f002125094de37d15e758eec54e751843509a/test/project/pkg/apis/ship/v1beta1/register.go.

Reply:
This is was the listers look for by default.

Reply:
wait, my mistake, I don't need this.